drill.cpp

```diff
@@ -847,12 +847,16 @@ map<int, drillbit> ExcellonProcessor::optimize_bits() {
                     return a.difference(wanted_length, inputFactor).value_or(std::numeric_limits<double>::infinity()) <
                         b.difference(wanted_length, inputFactor).value_or(std::numeric_limits<double>::infinity());
                 });
-            if (best_available_drill->difference(wanted_length, inputFactor)) {
+
+            auto difference = best_available_drill->difference(wanted_length, inputFactor);
```

Comment:
I prefer `const auto` if you don't mind.  That way I don't have to later wonder about whether or not it might change.

Reply:
Makes sense, thanks.